website/util/sanitize.py

```diff
@@ -14,6 +14,7 @@ def strip_html(unclean):
     return bleach.clean(unclean, strip=True, tags=[], attributes=[], styles=[])
 
 
+# TODO: Not used anywhere except unit tests? Review for deletion
```

Comment:
Good catch. I believe this function can be removed, but that can be in a separate PR.